client/src/components/Timetable/Viewer/TimetableViewer.js

```diff
@@ -120,13 +121,18 @@ export default function TimetableViewer() {
           height="parent"
           plugins={[timeGridPlugin, interactionPlugin]}
           weekends={hasWeekendClasses}
-          slotLabelFormat={{
-            hour: 'numeric',
-            minute: '2-digit',
-            omitZeroMinute: true,
-            hour12: false,
-            meridiem: 'narrow',
-          }}
+          slotLabelFormat={
```

Comment:
It's better for readability to pull out this conditional into a code block before the render:

```
let slotLabelFormat = undefined;
if (use24HourTime) { 
    slotLabelFormat = {...};
}
```

or simply `const slotLabelFormat = __ ? __ : __;`

The reason for this is to not mix business logic with rendering logic, especially across many lines.